tests/test_html_parser.py

```diff
@@ -28,3 +28,13 @@ def test_annotated_text():
 
     assert to_unicode(document.paragraphs[1].sentences[0]) == "Tento text je tu aby vyplnil prázdne miesto v srdci súboru."
     assert to_unicode(document.paragraphs[1].sentences[1]) == "Aj súbory majú predsa city."
+
+def test_from_url():
+    url = "https://it.wikipedia.org/wiki/Parsing"
+    LANG = "italian"
+    parser = HtmlParser.from_url(url, Tokenizer(LANG))
+    document = parser.document
+    test_str = "In informatica, il parsing, analisi sintattica o parsificazione è un processo che analizza un flusso continuo di dati in ingresso ( input, letti per esempio da un file o una tastiera) in modo da determinare la correttezza della sua struttura grazie ad una data grammatica formale."
```

Comment:
1. Without your fix, where would the empty space would be here? At the end of the sentence? 
2. Does this test fail without your fix? 
3. Can be the sentence shorter to highlight the fixed part?
4. Also, isn't the empty space in `n ingresso ( input, letti per esempio` not desirable too?

Reply:
@mrx23dot  try to parse the text example of the implemented test, without the fix.